spec/models/activerecord_obstacle_course_spec.rb

```diff
@@ -52,11 +53,11 @@
 
   it 'finds order id of smallest order' do
     # ----------------------- Using Raw SQL ----------------------
-    order_id = ActiveRecord::Base.connection.execute('SELECT id FROM orders ORDER BY amount ASC LIMIT 1').first['id']
+    # order_id = ActiveRecord::Base.connection.execute('SELECT id FROM orders ORDER BY amount ASC LIMIT 1').first['id']
     # ------------------------------------------------------------
 
     # ------------------ Using ActiveRecord ----------------------
-    # Solution goes here
+    order_id = Order.minimum(:id)
```

Comment:
This just happens to work, but isn't the same as the SQL query above. What is the query doing with regards to the order of the data, in order to get the order_id?